text/0002-bundler-2.md

```diff
@@ -0,0 +1,44 @@
+# Bundler 2 RFC
+
+This document outlines the changes proposed for the Bundler v2 release. The main goal of this release will be to remove
+support for Ruby < 2.0, remove and deprecate existing functionality and introduce any changes to existing behaviour.
+
+## Removed Functionality
+* The `bundle viz` and `bundle show` commands. The `viz` command will be moved to a Gem/Bundler Plugin and `show` will be removed in favour of `bundle info` and `bundle list`
```

Comment:
@colby-swandale Thank you for creating RFC document. It is very easy to understand.

I found "`` use `bundle show` instead of `bundle list` ``" which is a deprecation warning.

https://github.com/bundler/bundler/blob/v1.15.1/lib/bundler/cli.rb#L239

Below that there is "`` TODO: 2.0 remove `bundle list` ``".

https://github.com/bundler/bundler/blob/v1.15.1/lib/bundler/cli.rb#L243

Is it expected that both `bundle list` and` bundle show` should be removed, and instead use `bundle info`? Or remove `bundle list` and use `bundle show` is correct?